fern/apis/version-2024-11-13/definition/stt.yml

```
@@ -1,5 +1,219 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/fern-api/fern/main/fern.schema.json
 
+service:
+  base-path: /stt
+  auth: true
+  endpoints:
+    transcribe:
+      path: ""
+      method: POST
+      display-name: Speech to Text (Batch)
+      docs: |
+        Transcribes audio files into text using Cartesia's Speech-to-Text API.
+        
+        Upload an audio file and receive a complete transcription response. Supports audio files up to several hours in length with automatic intelligent chunking for longer audio.
```

cc @shreygupta2809 have we tested this out?

```suggestion
        Upload an audio file and receive a complete transcription response. Supports arbitrarily long audio files with automatic intelligent chunking for longer audio.
```